### ISSUE TITLE
Can't generate JP2 on large TIFF

### ISSUE THREAD

**Issue by eliotjordan** [pulibrary/figgy#4178]
From Beth Hass:

```
I’m attempting to upload an enormous file of a scroll and it seems to be hung up. I saved it as LZW compressed TIF to accommodate the 6.3GB file size. Thoughts? Do I need to size down?
```

https://figgy.princeton.edu/catalog/0854f072-18b0-412d-93d7-575e28682872 
https://app.honeybadger.io/projects/53391/faults/66345272

@tpendragon says we should maybe force the generation of pyramidal tiffs and skip the JP2 generation.

**Comment by tpendragon** [comment]
Attempting to generate the pyramidal tiff now. This might be our biggest scroll - we'll see how it goes.

**Comment by tpendragon** [comment]
For now I generated the pyramidal manually. The final fix will be #3940. The command to do this is the following:

```
resource_id = "4eae2e8a-5fd5-4792-bdea-8e2085465687"
change_set_persister = ::ChangeSetPersister.new(metadata_adapter: Valkyrie::MetadataAdapter.find(:indexing_persister), storage_adapter: Valkyrie::StorageAdapter.find(:pyramidal_derivatives))
VIPSDerivativeService.new(id: resource_id, storage_adapter: storage_adapter).create_derivatives
resource = change_set_persister.query_service.find_by(id: resource_id)
resource.processing_status = "processed"
change_set_persister.metadata_adapter.persister.save(resource: resource)
```
